src/mount.c

```diff
@@ -7,10 +7,13 @@
 #include <glib/gstdio.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <blkid.h>
+#include <libmount.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include "error.h"
 #include "mount.h"
+#include "utils.h"
```

Comment:
The corresponding header file `mount.h` should come last. Please move the include for `utils.h` above it.